website/src/pages/grading/grade-output.tsx

```diff
@@ -0,0 +1,231 @@
+
+import { QuestionMarkCircleIcon } from '@heroicons/react/20/solid'
+
+
+export default function OutputDetail(): JSX.Element {
+  return <>
+    <div className=" p-6 h-full mx-auto bg-slate-100 text-gray-800">
+      
+      {/* Instrunction and Output panels */}
+      <section className="mb-8  lt-lg:mb-12 ">
+        <div className="grid lg:gap-x-12 lg:grid-cols-2">
+          
+          {/* Instruction panel */}
+          <div className="rounded-lg shadow-lg h-full block bg-white">
+            <div className="p-6">
+              <h5 className="text-lg font-semibold mb-4">Instruction</h5>
+              <div className="bg-slate-800 p-6 rounded-xl text-white whitespace-pre-wrap">
+                {SAMPLE_PROMPT}
+              </div>
+            </div>
+          </div>
+
+          {/* Output panel */}
+          <div className="mt-6 lg:mt-0 rounded-lg shadow-lg h-full block bg-white">
+            <div className="p-6">
+              <h5 className="text-lg font-semibold mb-4">Output</h5>
+              <div className="bg-slate-800 p-6 rounded-xl text-white whitespace-pre-wrap">
+                {SAMPLE_OUTPUT}
+              </div>
+            </div>
+            {/* Form  wrap*/}
+            <div className="p-6">
+                <h3 className="text-lg text-center font-medium leading-6 text-gray-900">Rating</h3>
+                <p className="text-center mt-1 text-sm text-gray-500">
+                  (1 = worst, 7 = best)
+                </p>
+              
+            
+                {/* Rating buttons */}
+                <div className="flex justify-center p-6">
+                  <button
+                    type="button"
+                    className="inline-flex items-center mx-2 rounded-md border border-gray-300 bg-white px-4 py-2 text-sm font-medium text-gray-700 shadow-sm hover:bg-gray-50 focus:outline-none focus:ring-2 focus:ring-indigo-500 focus:ring-offset-2"
```

Comment:
(I have to get used to the repetitive tailwind style ..), Do you think it makes sense to create react-components for elements that appear multiple times to have the style at a central place?

Reply:
Yeah good idea. I had a button component but it disappeared. Let me try to at least wrap this up into a function in a few minutes. And at some point in the near future we can add a better common button component.

Yeah tailwind is a little odd looking at first with some of these super long class strings. I didn't like it at first mainly for this reason, but I'm a convert now. It definitely encourages componentization to address this, which is also a net positive.